org.libretro.RetroArch.json

```diff
@@ -59,7 +60,30 @@
               "sha256": "0f4620e11fa283fedafb474427c8e96bf149511a1804bdc47350963ae5cf54d8"
             }
           ]
-        }
+        },
+	{
+	  "name": "nvidia-cg-tookit",
```

Comment:
This is great! Was wondering why it wasn't working.

Mind switching the whitespace from tabs over to spaces? Thanks!

![n7azeef](https://user-images.githubusercontent.com/25086/28635027-221c7eea-7207-11e7-8f59-1ff7497dbab3.gif)